services/dqa-secondary-reports/README.md

```diff
@@ -0,0 +1,27 @@
+# Service for DQA secondary report information
+
+Service that exposes a endpoint for fetching PEDSnet DQA secondary report information for different data model versions, e.g.
+
+```
+/pedsnet/2.0.0
+/pedsnet/1.0.0
+```
+
+## Installation
+
+To install the prerequisites, run
+```
+pip3 install -r requirements.txt 
+```
+
+## Run
```

Comment:
This should just show how to get the help message:

```bash
python3 main.py -h
```

and an example of running it:

```bash
python3 main.py --token=abc123
```